core/src/main/kotlin/apiDocs/ProductsDocs.kt

```diff
@@ -483,7 +484,7 @@ val getVulnerabilitiesAcrossRepositoriesByProductId: OpenApiRoute.() -> Unit = {
 
 val getOrtRunStatisticsByProductId: OpenApiRoute.() -> Unit = {
     operationId = "GetOrtRunStatisticsByProductId"
-    summary = "Get statistics about ORT runs across the repositories of a product."
+    summary = "Get statistics about ORT runs across the repositories of a product"
```

Comment:
Sometimes we say "ORT runs" and sometimes just "runs", I think we should align on the latter.